draft-ietf-dance-architecture.md

```diff
@@ -465,7 +469,8 @@ A zone will have to manage a large amount of changes as devices are constantly a
 
 In these cases it is important to consider the architecture of the DNS zone and when possible use a tree-like structure with many subdomain parts, much like reverse DNS records or how telephone numbers are represented in the ENUM standard (RFC 6116).
 
-If an authoritative resolver were configured to respond quite slowly (think slow loris [XXXrefereceXXX]), is it possible to cause a DoS on the TLS server via complete exhaustion of TCP connections?
+If an authoritative resolver were configured to respond quite slowly using TCP, (for instance, a
+{{slowlaris}} attack), it possible that this would cause a TLS server to exhaust all it's TCP sockets.
```

Comment:
```suggestion
If an authoritative resolver were configured to respond quite slowly using TCP, (for instance, a
{{slowloris}} attack), it possible that this would cause a TLS server to exhaust all it's TCP sockets.
```